host-interaction/gui/enumerate-gui-resources.yml

```diff
@@ -18,3 +18,4 @@ rule:
       - api: EnumWindowStations
       - api: EnumDesktops
       - api: EnumWindows
+      - property: System.Windows.Forms.Screen::AllScreens
```

Comment:
Don't forget to update the `authors` field.

Reply:
Noted. https://github.com/mandiant/capa-rules/pull/601/commits/b3cd0abc6de717aaedda04b51804f1c562c8b472 updates the field.